packages/editor/src/components/block-inspector/index.js

```diff
@@ -79,15 +78,17 @@ const BlockInspector = ( { selectedBlock, blockType, count, hasBlockStyles } ) =
 
 export default withSelect(
 	( select ) => {
-		const { getSelectedBlock, getSelectedBlockCount } = select( 'core/editor' );
+		const { getSelectedBlockClientId, getSelectedBlockCount, getBlockName } = select( 'core/editor' );
 		const { getBlockStyles } = select( 'core/blocks' );
```

Comment:
I think we should use getBlockType from select ( 'core/blocks' ); instead of the wp.blocks one, otherwise when there are updates to the blocks our withSelect function may not be executed.
This may be related to the reason end 2 end tests were failing without the blockType check.

Reply:
I agree. I think we should avoid relying on `getBlockType`  (the global) entirely  but I didn't want to create inconsistency between components now. This should be done as a dedicated PR at some point.